### ISSUE TITLE
Change elements: change writer, penciller, inker, colorist, letterer, coverArtist, editor, translator, publisher and imprint maxOccurs to -1

### ISSUE THREAD

**Issue by Kara-Zor-El** [anansi-project/comicinfo#44]
### Where does this comes from?

I have not discussed this elsewhere but I am a developer of a open source eReader ([JellyBook](https://github.com/Kara-Zor-El/JellyBook)) who is looking to add proper ComicInfo support to my app

### What is the rationale for adding support for this element?

Some comics may be worked on by a team of people where there is a team of each group who may work on the same comic.

### Is the element already handled by any application or tool?

Some books May do:
`<Writer>Person 1, Person 2</Writer>` or such but it makes it unclear as all of the previous said categories are singular rather than plural entries

**Comment by gotson** [comment]
This is not gonna happen, as it would be breaking previous implementations. We know it's not ideal, but keeping compatibility is best.

**Comment by ajslater** [comment]
fwiw @Kara-Zor-El, there are some fledgling efforts to create a new comic metadata standard that is not so ugly and limited. Current discussions are centered around Readium, [OPDS v2.0](https://drafts.opds.io/opds-2.0) and the [DiViNa media extension](https://readium.org/webpub-manifest/profiles/divina).

ComicInfo.xml is probably not the long term future of comic metadata, but may receive non-breaking changes to be useful in the short term.

**Comment by gotson** [comment]
Indeed, [Readium Packaging Format](https://github.com/readium/webpub-manifest/blob/master/packaging.md) might be the future we need.
